ca_gazebo/launch/create_camera_world.launch

```diff
@@ -0,0 +1,32 @@
+<launch>
+  <param name="create1/amcl/initial_pose_x" value="0"/>
+  <param name="create1/amcl/initial_pose_y" value="4"/>
+  <param name="create1/amcl/initial_pose_a" value="0"/>
+  
+  <arg name="env" default="camera_world"/>
+
+  <arg name="paused" value="$(optenv PAUSED false)"/>
+  <arg name="gui" value="$(optenv GUI true)"/>
+  <arg name="debug" value="$(optenv DEBUG false)"/>
+  <env name="GAZEBO_MODEL_PATH" value="$(find ca_gazebo)/models"/>
+
+	<!-- Launch Gazebo world -->
```

Comment:
Why are these lines here? This launch file doesn't follow the same structure as the others.